internal/pkg/describe/service.go

```diff
@@ -82,20 +81,21 @@ type apprunnerClient interface {
 	DescribeService(svcArn string) (*apprunner.Service, error)
 }
 
-type apprunnerSvcDescriber interface {
+type ecsDescriber interface {
 	Params() (map[string]string, error)
+	Outputs() (map[string]string, error)
+	Platform() (*awsecs.ContainerPlatform, error)
+	EnvVars() ([]*awsecs.ContainerEnvVar, error)
+	Secrets() ([]*awsecs.ContainerSecret, error)
 	ServiceStackResources() ([]*stack.Resource, error)
-	Service() (*apprunner.Service, error)
-	ServiceARN() (string, error)
-	ServiceURL() (string, error)
 }
 
-type ecsStackDescriber interface {
+type apprunnerStackDescriber interface {
```

Comment:
Since the `Service`, `ServiceURL` methods use the `appRunnerClient` should we rename the interface?
```suggestion
type appRunnerDescriber interface 
```